### ISSUE TITLE
API api/2/repository/commit_resolution fails to commit resolution on deleted files

### ISSUE THREAD

**Issue by lolo782374** [craftercms/craftercms#5729]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the develop branch
- [X] The issue is in the latest released 4.1.x
- [X] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

API `api/2/repository/commit_resolution` shows no change after committing resolution to a deleted file in project sandbox. Committing resolution is successful only when the file had been added/modified. 

### Steps to reproduce

Steps:

1. Create new project in Studio.
2. Create new content in Pages sidebar.
3. Delete created content in filesystem. (not using git to commit this delete)
4. In `Project Tools->Git->Repository Status` attempt to commit resolution on deleted files.

Commit does not go through as expected, API returns 200 (OK), Studio UI shows **Successfully committed** as seen in the attached screenshot.

### Relevant log output

```shell
The git commit resolution shows only the commit resolution message (opijk). Deleted file not showing as part of the commit.

hi@DESKTOP-4J0OP3V:~/devupdate/craftercms/crafter-authoring/data/repos/sites/juion/sandbox$ git show b90c35db66f3ea3d9e5b5d2305b27e0edfda4965
commit b90c35db66f3ea3d9e5b5d2305b27e0edfda4965 (HEAD -> master)
Author: admin admin <evaladmin@example.com>
Date:   Thu Aug 11 13:11:37 2022 -0400

    opijk
```

### Screenshots and/or videos

<img width="1836" alt="image" src="https://user-images.githubusercontent.com/110685450/184196402-3f2883b1-2018-4086-8fe9-56e5b945ccb7.png">

**Comment by sumerjabri** [comment]
@Euquimides, please investigate and update the ticket. Please add to automation if not there already.

**Comment by Euquimides** [comment]
Hi @sumerjabri ! Regarding this ticket, I confirm the issue is still present in v.4.0.x, v4.1.x and v4.2. I updated the ticket accordingly.

**Comment by Euquimides** [comment]
Hi! Updated and rechecked this issue. Fix verified for v4.1.6 and v4.2. Thank you! Closing this ticket.
